### PR TITLE
Split doc

### LINKED ISSUE #278: which is the default projection ?
### Setup Information

- figanos version:
- Python version:
- Operating System:

### Description

In the docs, it says that Lambert Conformal is the default (this is also what I thought it was), but the actual default is PlateCarree.

@sarahclaude which one do we want ? I am leaning for Lambert.

### Steps To Reproduce

_No response_

### Additional context

_No response_

### Contribution

- [X] I would be willing/able to open a Pull Request to address this bug.

### PR DESCRIPTION
**Pull request by juliettelavoie**
### Pull Request Checklist:
- [x] This PR addresses an already opened issue (for bug fixes / features)
  - This PR fixes #278, #274, #202 
- [x] (If applicable) Documentation has been added / updated (for bug fixes / features).
- [ ] (If applicable) Tests have been added.
- [x] CHANGELOG.rst has been updated (with summary of main changes).
  - [x] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added.

### What kind of change does this PR introduce?

* Creates directory datasets with the data for plot hatch map that was too long to calculate on the fly 
* Splits Getting started and Colours of Figanos in 3 new notebooks to increase speed of RtD.
* Change default projection to LambertConformal
* Fix bug in projection for scattermap
* Create gallery wall

### Does this PR introduce a breaking change?
New default projection

### Other information:

1. I name the new directory datasets because data already exist with info for the colours.
2. I only put the problematic data there. Let me know if I should put more.
3. Let me know if there are some examples that I left out of the Gallery walll that you would want me to add.
4. If you know a way to hide the last line a cell, let me know. I would prefer to hide the `plt.savefig` but I haven't been able to make it work.

### CODE REVIEW

**Comment by Zeitsperre** [comment]
@juliettelavoie

This is looking great!

The docs are failing for an unrelated reason (lack of a UserAgent in our request calls). The fix needs to be determined first before we can build docs on RTD. Don't spend time on that.

**Review by Zeitsperre — APPROVED** [review]
The new documentation with gallery looks incredible! Well done!

I'm wondering though what the problem is with having a `/data` in the top-level here. Is there some function that already creates this folder on install?

**Comment by juliettelavoie** [comment]
> The new documentation with gallery looks incredible! Well done!
> 
> I'm wondering though what the problem is with having a `/data` in the top-level here. Is there some function that already creates this folder on install?

There is already a `data` folder in `src/figanos` that has other types of data (IPCC colormaps). I thought it would lead to mix ups if they were named the same thing even if they are not at the same level.

File: docs/notebooks/figanos_maps.ipynb [hunk 1]
```diff
@@ -0,0 +1,498 @@
...
+    "# inf_5 = chng_f.where(chng_f<0.5).to_dataset()\n",
+    "\n",
+    "# ens_stats = ensembles.ensemble_mean_std_max_min(ens)\n",
+    "\n",
+    "# out=ens_stats.prcptot_mean.mean(dim='time', keep_attrs='True').to_dataset()\n",
+    "# xs.save_to_netcdf(out, '../../datasets/hatchmap-ens_stats.nc')\n",
+    "# xs.save_to_netcdf(sup_8, '../../figanos/datasets/hatchmap-sup_8.nc')\n",
+    "# xs.save_to_netcdf(inf_5, '../../figanos/datasets/hatchmap-inf_5.nc')"
+   ]
+  },
```
[the remark this answers is not in the document]
Comment (juliettelavoie) [reply docs/notebooks/figanos_maps.ipynb:388]: It is useful for us to remember how we created the data. Note that it is hidden in the RtD.

File: docs/notebooks/figanos_maps.ipynb [hunk 2]
```diff
@@ -0,0 +1,498 @@
...
+     "execute": "never"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "\n",
+    "ens_stats = xr.open_dataset('../../datasets/hatchmap-ens_stats.nc').prcptot_mean\n",
+    "sup_8 = xr.open_dataset('../../datasets/hatchmap-sup_8.nc').changed\n",
+    "inf_5 = xr.open_dataset('../../datasets/hatchmap-inf_5.nc').changed\n",
+    "\n",
```
[the remark this answers is not in the document]
Comment (juliettelavoie) [reply docs/notebooks/figanos_maps.ipynb:405]: not sure I understand the comment ?

**Comment by juliettelavoie** [comment]
@Zeitsperre  I added the test dat in data as we discussed. I will let you change the config for pooch. After that I just use fetch to access it ?

**Comment by Zeitsperre** [comment]
@juliettelavoie Yes, exactly. I'll set things up so that the testing data is only fetched when running the notebooks and skipped on installation. Thanks again for your patience!

**Comment by juliettelavoie** [comment]
maybe time to come back to the "if it ain't broke" method :P

**Comment by Zeitsperre** [comment]
@juliettelavoie I wish there was an emoji for this:

image

File: docs/notebooks/figanos_maps.ipynb [hunk 3]
```diff
@@ -0,0 +1,493 @@
+{
```
Comment (juliettelavoie) [inline docs/notebooks/figanos_maps.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #45.    # plt.savefig("images/station_map.png", bbox_inches='tight')
</pre><p>mais les images ne sont pas save  dans images mais dans _static/_gallery, non ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/figanos/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (Zeitsperre) [reply docs/notebooks/figanos_maps.ipynb:1]: They are, but if we were to enable this and then build the docs, we would see the same warnings as earlier. This line is only useful for developers.

Would it make sense to place these in their own hidden cells?

[the remark this answers is not in the document]
Comment (Zeitsperre) [reply docs/notebooks/figanos_maps.ipynb:1]: The metadata field is explicitly preserved, yet the cell is not hidden? Am I missing something?

Comment (Zeitsperre) [reply docs/notebooks/figanos_maps.ipynb:1]: Done!

[the remark this answers is not in the document]
Comment (Zeitsperre) [reply docs/notebooks/figanos_maps.ipynb:1]: I was missing the step where I "save" the changes, haha. Done.

Comment (juliettelavoie) [reply docs/notebooks/figanos_maps.ipynb:1]: if it is in its own cell it doesn't save the figure from the previous cell. It saves an empty figure.

**Pull request merged**